src/content/developers/docs/apis/javascript/index.md

```diff
@@ -286,3 +290,4 @@ _Know of a community resource that helped you? Edit this page and add it!_
 
 - [Set up Web3js to use the Ethereum blockchain in Javascript](/developers/tutorials/set-up-web3js-to-use-ethereum-in-javascript/) _– Instructions for getting web3.js set up in your project._
 - [Calling a smart contract from JavaScript](/developers/tutorials/calling-a-smart-contract-from-javascript/) _– Using the DAI token, see how to call contracts function using JavaScript._
+- [Sending transactions using web3 and Alchemy](https://ethereum.org/en/developers/tutorials/sending-transactions-using-web3-and-alchemy/) _– Step by step walkthrough for sending transactions from the backend._
```

Comment:
```suggestion
- [Sending transactions using web3 and Alchemy](/developers/tutorials/sending-transactions-using-web3-and-alchemy/) _– Step by step walkthrough for sending transactions from the backend._
```

Reply:
FYI @elanh no need for root domain or language path when creating relative links.